### PR TITLE
+5 Icons

### PR DESCRIPTION
**Pull request by airon-main**
# Description

I've been trying lawnchair and then lawnicons. i was amazed by how simple yet incredible the apps are. so i tried making so much icons, turns out the guideline is outdated. so i remake some of the icons and make a new one

### Added
Get Current Wallpaper (`dk.appdictive.getcurrentwallpaper`)
PC Remote (`com.monect.portable`)
Quizizz (`com.quizizz_mobile`)
Milk Choco (`com.gameparadiso.milkchoco`)
Screen Dimmer (`com.github.ericytsang.screenfilter.app.android`)

## Contributor's checklist

- [x] I made the icons following [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

**Comment by airon-main** [comment]
This is my first time contributing. What are the checks not successful about?. I've run the app and it runs completely fine

File: app/assets/appfilter.xml
Comment (x9136) [inline app/assets/appfilter.xml:1]: >  What are the checks not successful about?

It's worth to add a space between `"` and `/>` everywhere.

File: svgs/pc_remote.svg
Comment (x9136) [inline svgs/pc_remote.svg:1]: Wrong size (`158`×109px).

File: svgs/screen_dimmer.svg
Comment (x9136) [inline svgs/screen_dimmer.svg:1]: Screenshot 2024-02-14 at 17 36 29
Screenshot 2024-02-14 at 17 37 40

**Review by x9136 — CHANGES_REQUESTED** [review]

File: svgs/pc_remote.svg
Comment (airon-main) [reply svgs/pc_remote.svg:1]: image
Too large maybe?

File: app/assets/appfilter.xml
Comment (airon-main) [reply app/assets/appfilter.xml:1]: just committed 👍

File: svgs/screen_dimmer.svg
Comment (airon-main) [reply svgs/screen_dimmer.svg:1]: image

like this?

Comment (airon-main) [reply svgs/screen_dimmer.svg:1]: image

like this?

Comment (airon-main) [reply svgs/screen_dimmer.svg:1]: image

File: svgs/pc_remote.svg
Comment (airon-main) [reply svgs/pc_remote.svg:1]: image

maybe you mean like this

Comment (x9136) [reply svgs/pc_remote.svg:1]: Yes, the long side should be 160px.

File: svgs/screen_dimmer.svg
Comment (x9136) [reply svgs/screen_dimmer.svg:1]: Screenshot 2024-02-15 at 09 25 51
If you want to round all corners, then don't forget to resize the shape after that.

image
It's enough to round the selected corners by 6px, though.

**Review by x9136 — CHANGES_REQUESTED** [review]

**Comment by airon-main** [comment]
done commit on the screen dimmer :))

File: svgs/screen_dimmer.svg
Comment (airon-main) [reply svgs/screen_dimmer.svg:1]: Done commit 👍

Comment (airon-main) [reply svgs/screen_dimmer.svg:1]: Sorry i had some school projects

**Pull request merged**